<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>파일 백업 프로그램</string>
  </property>
  <widget class="QGroupBox" name="groupSelectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>파일 선택</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>372</width>
      <height>319</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxAll">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>전체선택</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox1">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string> 한글파일, 텍스트파일(hwp, txt)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox2">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>엑셀파일(xls, xlsx, xlsm, xlm)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox3">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>그림파일(png, jpg, bmp, ..)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox4">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>PDF파일</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox5">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>프로그램 언어 소스(py, c, cpp, cs, java)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox6">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>동영상파일(avi, mpg, mp4, kmv, mkv)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="checkBox7">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>포토샵, 일러스트(psd, ai)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="checkBox8">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>압축파일(zip, rar, alz)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>백업 폴더</string>
   </property>
   <widget class="QLineEdit" name="lineEditSrc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C:/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSrc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더선택</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>대상 폴더</string>
   </property>
   <widget class="QLineEdit" name="lineEditDest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C:/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더선택</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>백업 시작</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>851</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
